agent/address.h

```diff
@@ -60,6 +60,7 @@
 #include <sys/socket.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
+#include <netdb.h>
```

Comment:
> switching to cmake is not something we're going to accept as maintainers. It will be less effort, and less churn, to stay with autotools and just ensure the `.sln` files are up to date.

That’s still true. I’m not going to merge cmake changes, but will happily merge any updates to the `.sln` files if you want to produce them. :-)